site/content/en/docs/Guides/access-api.md

```diff
@@ -236,7 +293,6 @@ $ curl http://localhost:8001/apis/stable.agones.dev/v1alpha1/namespaces/default/
 
 # allocate a gameserver from a fleet named 'simple-udp', with GameServerAllocation
 
-{{% feature expiryVersion="0.12.0" %}}
```

Comment:
This looks a bit funny, but I split all of the new stuff into a different block so that it's easier to see what changed.